pkg/controller/migration/convert/core.go

```diff
@@ -71,6 +78,76 @@ func handleCore(c *components, install *Installation) error {
 		install.Spec.FlexVolumePath = "None"
 	}
 
+	// Volumes for lib-modules, xtables-lock, var-run-calico, var-lib-calico, or policysync have been changed
+	v := getVolume(c.node.Spec.Template.Spec, "lib-modules")
+	if v == nil || v.HostPath == nil || v.HostPath.Path != "/lib/modules" {
+		return ErrIncompatibleCluster{"must mount /lib/modules from host"}
```

Comment:
Should this message be more like `expected calico-node to have Volume 'lib-modules' with host path '/lib/modules'`? (same for the others below too)